tests/conftest.py

```diff
@@ -2670,6 +2670,14 @@ def flow_cell_directory_name_demultiplexed_with_bcl_convert(
     return f"230504_A00689_0804_B{flow_cell_name_demultiplexed_with_bcl_convert}"
 
 
+@pytest.fixture
+def flow_cell_directory_name_demultiplexed_with_bcl_convert_flat(
```

Comment:
For fixture we try to always define a name, scope and for the actual function name it `fixture_<chosen name>`
```suggestion
@pytest.fixture(name="flow_cell_directory_name_demultiplexed_with_bcl_convert_flat",scope="session)
def fixture_flow_cell_directory_name_demultiplexed_with_bcl_convert_flat(
```